### PR TITLE
Improved OpAmp-Examples, new partial FourPin necessary

### PR DESCRIPTION
**Pull request by AHaumer**
Improvements of Electrical.Analog.Examples.OpAmps:
* impemented subpackages "OpAmpCircuits" with ready-to-use NonInvertingAmplifier, InvertingAmplifier, Adder, Subtracter, Differenatiator, Integrator, LowPass, HighPass, PI
* Using these in the Examples instead of implementing the circuit in the example
* New Examples: Adder, Subtracter, ControlCircuit (with PI)
Necessary partial FourPin implemented in Electrical.Analog.Interfaces.
For consistency:
Electrical.Analog.Interfaces.TwoPort extends from FourPin.
Electrical.Analog.Interfaces.OnePort extends from TwoPin.

### CODE REVIEW

**Comment by christiankral** [comment]
Proposing different `OCVmin` (start) values in ahaumer/ModelicaStandardLibrary#1

File: Modelica/Electrical/Analog/Examples/OpAmps.mo [hunk 1]
```diff
@@ -939,4 +1005,491 @@ package OpAmps "Examples with operational amplifiers"
         Tolerance=0.0001,
         Interval=1e-005));
   end LCOscillator;
+
+  package OpAmpCircuits "Different curcuits with operational amplifiers"
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Examples/OpAmps.mo]:
```suggestion
  package OpAmpCircuits "Different circuits with operational amplifiers"
```

**Comment by christiankral** [comment]
> Proposing different `OCVmin` (start) values in [AHaumer#1](https://github.com/AHaumer/ModelicaStandardLibrary/pull/1)

I was referencing to the wrong issue: ahaumer/ModelicaStandardLibrary#2 is the right ticket

File: Modelica/Electrical/Analog/Examples/OpAmps.mo (hunk 1 above)
Comment (AHaumer) [reply Modelica/Electrical/Analog/Examples/OpAmps.mo]: done

**Pull request merged**